GitUI/GitModuleForm.cs

```diff
@@ -61,11 +63,41 @@ protected GitModuleForm()
             }
         }
 
+        [DllImport("dwmapi.dll")]
+        private static extern int DwmSetWindowAttribute(IntPtr hwnd, int attr, ref int attrValue, int attrSize);
```

Comment:
This may be useful: https://docs.microsoft.com/en-us/dotnet/standard/native-interop/best-practices#common-windows-data-types
```suggestion
        private static extern int DwmSetWindowAttribute(IntPtr hwnd, uint attr, ref int attrValue, uint attrSize);
```